### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by Drapegnik**
[babajka-markup/storybook](https://babajka.github.io/babajka-markup/storybook)

## storybook ui
screen shot 2019-01-08 at 01 18 15

## new commands
```bash
npm run storybook
npm run storybook:build
```

## how to render `ejs` in stories:

1. simple `.ejs` template:
```js
// foo.stories.js

import render from './foo.ejs';
storiesOf('Foo', module).add('foo', ({ parameters: { defaultData } }) => render(defaultData))
```

2. `.ejs` with `include()` statements:

```ejs
<!-- parent.ejs -->

<%- include('./childA.ejs', { foo: 'bar' }) %>
<%- include('./childB.ejs', { bar: 'baz' }) %>
```

```js
// parent.stories.js

import parent from './parent.ejs';
import childA from './childA.ejs';
import childB from './childB.ejs';

const render = withIncludes({ './childA.ejs': childA, './childB.ejs': childB });

storiesOf('Parent', module).add('default', ({ parameters: { defaultData } }) => render(defaultData))
```

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<html>
```
Comment (UladBohdan) [inline static/index.html]: This file redirects from `localhost:3001` to github pages

File: bin/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```
Comment (UladBohdan) [inline bin/deploy.sh]: is this to avoid mess is `gh-pages` branch?

File: src/index.scss [hunk 3]
```diff
@@ -8,5 +8,5 @@
 @import 'font-awesome';
 
 // Fonts.
-@import 'https://fonts.googleapis.com/css?family=Comfortaa|Cormorant|Open+Sans';
+@import url('https://fonts.googleapis.com/css?family=Comfortaa|Cormorant|Open+Sans');
```
Comment (UladBohdan) [inline src/index.scss]: I think there's no need to import `Open Sans` twice.

File: static/index.html (hunk 1 above)
Comment (UladBohdan) [reply static/index.html]: That is still an issue!

**Comment by Drapegnik** [comment]
[add storybook on markup](https://trello.com/c/2MbMxu0B/319-add-storybook-on-markup)

File: bin/deploy.sh (hunk 2 above)
Comment (Drapegnik) [reply bin/deploy.sh]: rather to avoid mess in `package.json` because `deploy` command grew too much

**Pull request merged**